project_stage_state/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# GNU Affero General Public License <http://www.gnu.org/licenses/>
+
+{
+    'name': 'Add State field to Project Stages',
+    'version': '12.0.1.0.0',
+    'category': 'Project Management',
+    'summary': 'Restore State attribute removed from Project Stages in 8.0',
+    'author': "Daniel Reis,Odoo Community Association (OCA)",
+    'website': 'https://github.com/OCA/project-service',
```

Comment:
This should be corrected to actual repository of the module

Reply:
@alexey-pelykh , Could you please check it now?